src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+const Big = require('big.js');
+
+const operate = (numberOne, numberTwo, operation) => {
+  if (numberOne === 'NaN' || numberTwo === 'NaN') {
+    return 'NaN';
+  }
+  const one = new Big(numberOne);
+  const two = new Big(numberTwo);
+  switch (operation) {
+    case '+':
+      return one.plus(two).toString();
+    case '-':
+      return one.minus(two).toString();
+    case 'x':
+      return one.times(two).toString();
+    case '÷':
+      try {
+        return one.div(two).toString();
+      } catch (error) {
+        return 'NaN';
+      }
```

Comment:
**REQUIRED**
The modifier operator has been added but the operation is not yet supported.